shared/constants/types/people.tsx

```diff
@@ -12,7 +12,11 @@ export const stringToItemID: (id: string) => ItemID = id => id
 export type TodoTypeEnum = RPCTypes.HomeScreenTodoType
 export type TodoType = keyof typeof RPCTypes.HomeScreenTodoType
 
-export type _TodoMetaEmail = {type: 'email'; email: string}
+export type _TodoMetaEmail = {
+  type: 'email'
+  email: string
+  lastVerifyEmailDate?: number
```

Comment:
`// in seconds`

Reply:
also, since this is getting `0` as a default value in `makeTodoMetaEmail`, might as well get rid of the `?` here